test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,160 @@
+        b.set_input_text("#network-wireguard-settings-publickey-peer-0", m2_pubkey)
+        b.set_input_text("#network-wireguard-settings-endpoint-peer-0", f"192.168.100.12:{m2_port}")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-0", m2_ip)
+        b.click("button:contains('Add peer')")
+        b.wait_visible("#network-wireguard-settings-peer-1")
+        b.set_input_text("#network-wireguard-settings-publickey-peer-1", m2_pubkey)
+        b.set_input_text("#network-wireguard-settings-endpoint-peer-1", f"192.168.100.12:{m2_port}")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-1", m2_ip)
+        b.click("button#network-wireguard-settings-btn-close-peer-0")
+        b.wait_not_present("#network-wireguard-settings-peer-1")
+
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-wireguard-settings-dialog")
+
+        m2.execute(f"wg set wg0 peer {m1_pubkey} allowed-ips {m1_ip}/32")  # endpoint and port is not necessary for a peer if that peer estalishes the connectio first (i.e. the client)
+
+        # check connection over ipv4
+        try:
+            m1.execute(f"ping -c 5 {m2_ip}")
+        except testlib.Error:
+            m1.execute("ip a >&2; ip route >&2; nmcli c >&2; wg >&2")
+            m2.execute("ip a >&2; ip route >&2; nmcli c >&2; wg >&2")
+            raise
+
+        # check connection over ipv6
+        m2.execute("ip addr add 2001::2/64 dev wg0")
+        m2.execute(f"wg set wg0 peer {m1_pubkey} allowed-ips {m1_ip}/32,2001::1")
+
+        b.click("#networking-interfaces button:contains('wg0')")
+        b.click("#networking-edit-wg")
+        b.wait_visible("#network-wireguard-settings-dialog")
+        b.set_input_text("#network-wireguard-settings-allowedips-peer-0", f"{m2_ip}/32,2001::2")
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-wireguard-settings-dialog")
+
+        b.click("#networking-edit-ipv6")
+        b.wait_visible("#network-ip-settings-dialog")
+        b.select_from_dropdown("#network-ip-settings-select-method", "manual")
+        b.set_input_text("#network-ip-settings-address-0", "2001::1")
+        b.set_input_text("#network-ip-settings-netmask-0", "64")
+        b.set_input_text("#network-ip-settings-gateway-0", "::")
+        b.click("button:contains('Save')")
+        b.wait_not_present("#network-ip-settings-dialog")
+        self.wait_for_iface_setting("IPv6", "Address 2001:0:0:0:0:0:0:1/64")
+
+        m1.execute("until ip a show dev wg0 | grep -q 'inet6 2001::1/64 scope global'; do sleep 0.3; done", timeout=10)
+
+        try:
+            m1.execute("ping -6 -c 5 2001::2")
+        except testlib.Error:
+            m1.execute("ip a >&2; ip -6 route >&2; nmcli c >&2; wg >&2")
+            m2.execute("ip a >&2; ip -6 route >&2; nmcli c >&2; wg >&2")
+            raise
+
+        # install wireguard-tools from the install dialog
+        if not m1.ostree_image:
```

Comment:
Please test the ostree case as well.  In particular, the behaviour for rhel4edge, where the wireguard-tools package is not installed, but also cannot be installed. The "Add VPN" button then either should not appear at all, or it should show an useful error message akin to "wireguard-tools package is not installed"